contrib_bots/lib/githubcomment.py

```diff
@@ -0,0 +1,130 @@
+# See readme.md for instructions on running this code.
+from github import GithubConn
+import requests
+import json
+
+
+class GitHubHandler(object):
+    '''
+    This plugin facilitates allows you to comment on a GitHub issue, under a certain repository.  It
+    looks for messages starting with '@comment' or '@github-comment.
+    '''
+
+    def usage(self):
+        return '''
+            This bot will allow users to comment on a GitHub issue.  Users should preface
+            messages with "@comment" or "@github-comment". You will need to have a GitHub account.
+
+            Before running this, make sure to get a GitHub OAuth token, and store it in the github_token.txt file.
+            Also, edit the path_to_token variable in the send_to_github function to be the path to your token.
+            Please input info like this: /<username>/<repository_owner>/<repository>/<issue_number>/<comment>
+            '''
+
+    def triage_message(self, message):
+        # return True iff we want to (possibly) response to this message
+
+        original_content = message['content']
+
+        is_comment = (original_content.startswith('@comment') or
+                      original_content.startswith('@github-comment'))
+
+        return is_comment
+
+    def handle_message(self, message, client, state_handler):
+        original_content = message['content']
+        original_sender = message['sender_email']
+
+        # this handles the message if its starts with @comment
+        if original_content.startswith('@comment'):
+
+            params = get_values_message(original_content)
+
+            status_code = send_to_github(params['user'], params['repo_owner'], params['repo'],
+                                         params['issue'], params['comment_body'])
+
+            if status_code == 201:
+                # sending info to github was successful!
+                reply_message = "You commented on issue number " + params['issue'] + " under " + \
+                                params['repo_owner'] + "'s repository " + params['repo'] + "!"
+
+                client.send_message(dict(
+                    type='private',
+                    to=original_sender,
+                    content=reply_message,
+                ))
+
+            else:
+                # sending info to github did not work
+                reply_message = ":( There was a problem commenting on issue number " + params['issue'] + " under " + \
+                                params['repo_owner'] + "'s repository " + params['repo'] + \
+                                ". Did you enter the information in the correct format?"
+
+                client.send_message(dict(
+                    type='private',
+                    to=original_sender,
+                    content=reply_message,
+                ))
+
+        # handle if message starts with @github-comment
+        if original_content.startswith('@github-comment'):
```

Comment:
Those conditions should be exclusive, meaning a message can start either with `@comment` or `@github-comment`, so this should be an `else if`.

The code is repeated in both `if` code blocks - could you make one function out of it you'll be calling once? There is no need for repeating the code.